server/src/middlewares/Authentication.ts

```diff
@@ -50,25 +50,49 @@ const handleConfidentialAuth = (
             name: decoded.name,
         };
 
-        if(!user || !user.name || !user.id) {
+        if (!user || !user.name || !user.id) {
             logger.warning({
                 service: Service.SERVER,
                 severity: Severity.HIGH,
                 workflow: 'Authentication',
                 step: 'got unauthorized token'
             })
-            return response.status(403).json({error: "forbidden prod user"});
+            return response.status(403).json({ error: "forbidden prod user" });
         }
 
         logger.info({
             service: Service.SERVER,
             severity: Severity.HIGH,
             workflow: 'Authentication',
-            step: `authorized azure token successfully! got user: ${JSON.stringify(user)}` 
-        })
-        response.locals.user = user;
-        response.locals.epidemiologynumber = request.headers.epidemiologynumber;
-        return next();
+            step: `authorized azure token successfully! got user: ${JSON.stringify(user)}`
+        });
+
+        logger.info({
+            service: Service.SERVER,
+            severity: Severity.LOW,
+            workflow: 'Authentication',
+            step: `request to the graphql API with parameters: ${user.id}`,
+            user: response.locals.user.id,
```

Comment:
```suggestion
            user: user.id,
```
because the response.locals.user is not initated yet

Reply:
fixed